crates/iron-remote-desktop/src/session.rs

```diff
@@ -54,6 +54,9 @@ pub trait SessionBuilder {
     #[must_use]
     fn canvas_resized_callback(&self, callback: js_sys::Function) -> Self;
 
+    #[must_use]
+    fn disable_nla(&self, disable: bool) -> Self;
```

Comment:
issue: You probably forgot to remove your modifications to the common interface here